.github/workflows/make-live.yml

```diff
@@ -3,6 +3,8 @@ name: Make Live
 
 # This workflow only runs when it is manually dispatched
 on:
+  push:
+    branches: [ jashapiro/s3-render ]
```

Comment:
I know this is designed to be run via `workflow_dispatch` so we can file a PR with the updated notebooks. Personally, I thought it would have taken longer to run the make live step than it does (less than 20 minutes for ~23 min total). Have you considered a separate workflow on `pull_request` that makes sure things _run_?

Reply:
Ah, I see:

> If we are satisfied with the way things are working, I can continue to work on this branch/PR, adding documentation in CONTRIBUTING.md and uncommenting the creation of a PR to check that the changes it makes are the ones we actually expect.

I think what you're saying accomplishes the same objective but differently.

Reply:
Yeah, I think I was thinking it would take too long to run on every PR, but that is absolutely an option. Given that it seems you can't set variables when making a PR (as I write that, I think we could do it with some cleverness, but probably not worth it) I would consider a separate action for the test and generate PRs tasks.

We could check building with every pull requests to master, and submit a PR with rendered/live updates on merge to master?

Reply:
> We could check building with every pull requests to master, and submit a PR with rendered/live updates on merge to master?

This plan is definitely in the _spirit_ of  #296, even if it is not exactly what I wrote there.

Reply:
Filed https://github.com/AlexsLemonade/training-modules/issues/359 - if we like this plan better (I think _I_ do because it doesn't require us to revise `scripts/make-live.R`), I will close #296 and #299.